app/models/agents/email_digest_agent.rb

```diff
@@ -29,22 +33,20 @@ def default_options
 
     def receive(incoming_events)
       incoming_events.each do |event|
-        self.memory['queue'] ||= []
-        self.memory['queue'] << event.payload
```

Comment:
Nice way to reduce the memory size. How about adding a migration to clear the memory `queue` of existing agents?
I think it would also be a good idea to add a validation which ensures that `keep_events_for` big enough to not delete the events before the agent is ran the next time.